### ISSUE TITLE
Recipient conditions cannot be saved

### ISSUE THREAD

**Issue by gogogorski** [verbb/formie#1013]
### Describe the bug

I try to configure recipient conditions for an email notification in a form. I'm setting a condition depending on a field value in a email notification in the content tab. After hitting 'apply' the condition is set, but after hitting 'save' on the form the condition is not saved.

Setting the condition before 'saving' the form: 
![Bildschirmfoto 2022-07-28 um 14 05 21](https://user-images.githubusercontent.com/51097187/181501212-1a60d13f-1678-4c98-92d7-f58d05393f63.png)

Then after 'saving' the condition is gone:
![Bildschirmfoto 2022-07-28 um 14 05 33](https://user-images.githubusercontent.com/51097187/181501282-64d776c0-d0d3-4266-aa1d-f9219cb6519b.png)

I've tried different fields and email addresses for the condition, but nothing works.

I'm using Craft v4.2.0.2 and Formie v2.0.4 (dev) and Craft v4.1.4.1 (production). Neither is working.

### Steps to reproduce

1. Create a new email notification on a form
2. Set recipient conditions on the email notification 
3. Hit 'apply' on the modal
4. 'Save' the form
5. Open the admin notification settings again

### Form settings

- Multi-page form: No
- Submission Method: Ajax or Page Reload (doesn't matter)
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.2.0.2

### Plugin version

2.0.4

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by gogogorski** [comment]
Thanks for the reply and fixing the issue. Do already know when the next release is going to happen?

**Comment by engram-design** [comment]
Should have a release out over the weekend

**Comment by engram-design** [comment]
Fixed in [2.0.5](https://github.com/verbb/formie/releases/tag/2.0.5)
